### PR TITLE
chore: bump version to 1.3.2

### PR DESCRIPTION
**Pull request by ogabrielides**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1,5 +1,5 @@
 AC_PREREQ([2.60])
-AC_INIT([libdashbls],[1.3.0])
+AC_INIT([libdashbls],[1.3.1])
```
Comment (UdjinM6) [inline configure.ac:2]: we have 1.3.1 already https://github.com/dashpay/bls-signatures/releases/tag/v1.3.1
```suggestion
AC_INIT([libdashbls],[1.3.2])
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: configure.ac (hunk 1 above)
Comment (ogabrielides) [reply configure.ac:2]: oups

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by shumkov — APPROVED** [review]
👍

**Comment by shumkov** [comment]
CI is failing. Is it intended?

**Pull request merged**